optuna/integration/wandb.py

```diff
@@ -51,15 +57,17 @@ def objective(trial):
                 x = trial.suggest_float("x", -10, 10)
                 return (x - 2) ** 2
 
+            study = optuna.create_study(study_name="my_study")
```

Comment:
ditto (https://github.com/optuna/optuna/pull/3403/files#r835770611)
```suggestion

            study = optuna.create_study(study_name="my_study")
```